### ISSUE TITLE
Fix order of witness receipts relative to delegation approval.

### ISSUE THREAD

**Issue by pfeairheller** [WebOfTrust/keripy#503]
In the current implementation of delegated inception and rotation (including in multisig delegation), the delegate controller waits for delegation approval before publishing the event to the witnesses.  Since the witnesses are under the control of the controller, this order should be inverted.

To accomplish this, witnesses should accept delegated AID events without requiring a delegator approval anchor but not publish the events externally until the delegator's approval anchor is received.  Once watchers are in place the witnesses can publish the event externally without the approval because the watcher's won't accept the event until the approval shows up.

**Comment by pfeairheller** [comment]
No, considering the issue you just opened, it is not complete for multisig delegation and needs work there, I believe.

**Comment by kentbull** [comment]
This is fixed by https://github.com/WebOfTrust/keripy/pull/1100. Once we get that merged then we can close this issue.
